src/tokens/allTokens.json

```diff
@@ -431,5 +431,1013 @@
+    "extensions": {
+      "parentAddress": "0x6cd775540cce9adb01872db9f794142f53595c71",
+      "project": {
+        "name": "Gistcoin Payment",
+        "summary": "Gistcoin is a utility token for all influencers, streamers, musicians, artists, brands, and content creators from all backgrounds who create content across social media communities. We believe that a decentralized token system allows for all creators - including creators from underrepresented groups - to have direct access to their fans and monetization methods.",
+        "contact": "hello@gistcoin.com",
+        "website": "gistcoin.io"
+      }
+    }
+  },
+  {
+    "chainId": 137,
+    "name": "Orbs Token",
+    "symbol": "ORBS",
+    "decimals": 18,
+    "address": "0x614389EaAE0A6821DC49062D56BDA3d9d45Fa2ff",
+    "logoURI": "https://raw.githubusercontent.com/orbs-network/graphical-assets/main/orbs/ethereum/token/logo256.png",
+    "tags": [
+      "pos",
+      "erc20"
+    ],
+    "extensions": {
+      "parentAddress": "0xff56Cc6b1E6dEd347aA0B7676C85AB0B3D08B0FA",
+      "project": {
+        "name": "Orbs Network",
+        "summary": "Orbs' advanced architecture puts it in a unique position to work in conjunction with EVM-based L1s and L2s by serving as a separate execution layer that runs decentralized applications that enrich the capabilities of EVM-based smart contracts.",
+        "contact": "https://discord.gg/sswGDYGBt5",
+        "website": "https://www.orbs.com/"
+      }
+    }
+  },
+  {
+    "chainId": 137,
+    "name": "InsurAce",
+    "symbol": "INSUR",
+    "decimals": 18,
+    "address": "0x8a0e8b4b0903929f47C3ea30973940D4a9702067",
+    "logoURI": "https://app.insurace.io/asset/token/insur.png",
+    "tags": [
+      "pos",
+      "erc20"
+    ],
+    "extensions": {
+      "parentAddress": "0x544c42fBB96B39B21DF61cf322b5EDC285EE7429",
+      "project": {
+        "name": "InsurAce",
+        "summary": "InsurAce is a decentralized insurance protocol, aiming to provide reliable, robust, and carefree DeFi insurance services to DeFi users, with a low premium and sustainable investment returns.",
+        "contact": "contact@insurace.io",
+        "website": "https://www.insurace.io/"
+      }
+    }
+  },
+  {
+    "chainId": 137,
+    "name": "MASQ",
+    "symbol": "MASQ",
+    "decimals": 18,
+    "address": "0xee9a352f6aac4af1a5b9f467f6a93e0ffbe9dd35",
+    "logoURI": "https://github.com/MASQ-Project/MASQ-contract/raw/master/MASQ%20Logo%20Blue%20Solo%20Transparent.png",
+    "tags": [
+      "pos",
+      "erc20"
+    ],
+    "extensions": {
+      "parentAddress": "0x06F3C323f0238c72BF35011071f2b5B7F43A054c",
+      "project": {
+        "name": "MASQ Network",
+        "summary": "The MASQ software offers users a true privacy web3.0 browsing experience, based on a peer to peer mesh network. Users can earn cryptocurrency by supporting the network, and provides Internet freedom for users worldwide.",
+        "contact": "https://discord.gg/masqo",
```

Comment:
@py-zoid Greetings there.

Thanks again for adding our token to the Polygon token list.

We note that there is a small typo on our discord link.

it should be:
https://discord.gg/masq

thanks!
KauriHero
Project Lead & Co-Founder